python/cudf/cudf/core/column/numerical.py

```diff
@@ -412,6 +412,61 @@ def is_monotonic_decreasing(self):
                 )
         return self._is_monotonic_decreasing
 
+    def overflow_safe_to(self, to_dtype):
```

Comment:
Maybe rename to `can_cast` (ala numpy), or perhaps `can_cast_safely`?

Reply:
👍